client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -48,6 +48,7 @@ import {
 	domainManagementTransferInPrecheck,
 } from 'calypso/my-sites/domains/paths';
 import { emailManagement } from 'calypso/my-sites/email/paths';
+import TitanSetUpThankYou from 'calypso/my-sites/email/titan-set-up-thank-you';
```

Comment:
I can't move it to a different place, linter complains about not being in alphabetical order :) 
![linting](https://user-images.githubusercontent.com/5689927/132379400-47e5dd74-9e90-47cc-8d4e-c1eab1f649bd.gif)

Reply:
Phew! I didn't realize there's been some changes since I was AFK. It looks like we now use the `import/order` rule as [explained here](https://calypsop2.wordpress.com/2021/08/27/import-order-in-calypso/), which rather groups each import based on its type and alphabetizes the import path instead.

So we are good here - my bad.